soc/arm/nordic_nrf/nrf54l/Kconfig.defconfig.nrf54l15soc1_cpuapp

```diff
@@ -0,0 +1,26 @@
+# Nordic Semiconductor nRF54L15 MCU
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF54L15_CPUAPP
+
+config SOC
+	string
+	default "nRF54L15_CPUAPP"
+
+config SOC_DOMAIN_NAME
+	string
+	default "application"
```

Comment:
@tejlmand is this something we should have?

Reply:
Removed `SOC_DOMAIN_NAME` config (not used anywhere).